src/index.html

```diff
@@ -5,8 +5,460 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link"
+            data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="product" data-qa="card">
+          <img
+          class="product__photo"
+          src="./images/imac.jpeg"
+          alt="APPLE iMac photo"
+          >
```

Comment:
The same as above, need to fix it everywhere
```suggestion
          <img
            class="product__photo"
            src="./images/imac.jpeg"
            alt="APPLE iMac photo"
          >
```